contrib/agent/src/integration-test/java/io/opencensus/contrib/agent/instrumentation/ThreadInstrumentationIT.java

```diff
@@ -114,7 +114,10 @@ public void execute(Runnable command) {
             // been wrapped in a different context (by automatic instrumentation of
             // Executor#execute), that context will be attached when executing the Runnable.
             Context context2 = Context.current().withValue(KEY, "wrong context");
-            context2.attach();
+            Context newContext = context2.attach();
+            if (newContext == null) { // work around findBugs.
```

Comment:
newContext is never used here so don't understand why this is a fingBug warning.

Reply:
`Context.attach()` is marked as `@CheckReturnValue` in gRPC v0.14, so if we don't check the return value of it, `findBugs` will fail.

Or do you think we should disable this check in findbugs-exclude.xml?

Reply:
Sounds good, I filed https://github.com/grpc/grpc-java/issues/4817 to keep track of this.